packages/flutter_tools/lib/src/base/analyze_size.dart

```diff
@@ -0,0 +1,336 @@
+import 'dart:convert';
+
+import 'package:flutter_tools/src/base/process.dart';
+import 'package:flutter_tools/src/base/terminal.dart';
+import 'package:intl/intl.dart';
+import 'package:meta/meta.dart';
+import 'package:vm_snapshot_analysis/treemap.dart';
+
+import '../base/file_system.dart';
+import '../globals.dart' as globals;
+
+// This file can be in devtools_shared
```

Comment:
this comment looks like a TODO.
Add a small top level
`///` doc comment for this class.

Reply:
Do we really want this to live in DevTools shared? It has some deps in flutter_tool, which would create a circular dependency. I think it is fine to stay in flutter_tools